cpovc_offline_mode/helpers.py

```diff
@@ -14,36 +14,38 @@ def get_ovc_school_details(ovc):
     if ovc.school_level == "SLNS":
         return {}
 
-    try:
-        school = OVCEducation.objects.get(person_id=ovc.person.id, is_void=False)
-        return {
-            'school_name': school.school.school_name,
-            'school_class': school.school_class,
-            'admission_type': school.admission_type
-        }
-    except OVCEducation.DoesNotExist as e:
+    schools = OVCEducation.objects.filter(person_id=ovc.person.id, is_void=False)[:1]
```

Comment:
Safe guard against `MultipleObjectsReturned` error just in case multiple  non void `ovc_education` exists per person_id